src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/Organisation/OrganisationService.cs

```diff
@@ -167,4 +169,27 @@ public async Task<CheckRegulatorOrganisationExistResponseModel> GetRegulatorOrga
 
         return result;
     }
+
+    public async Task<OrganisationRelationshipModel> GetOrganisationRelationshipsByOrganisationId(Guid organisationExternalId)
+    {
+        HttpResponseMessage result = null;
+        var endpoint = $"{OrganisationGetSubsidiaryUri}/{organisationExternalId}/organisationRelationships";
+        var response = await _httpClient.GetAsync(endpoint);
```

Comment:
This should be inside the try, but it is also a duplicate call with response below

Reply:
Refactored